paddle/phi/kernels/cpu/embedding_kernel.cc

```diff
@@ -48,6 +48,11 @@ struct EmbeddingCPUFunctor {
     dev_ctx_.template Alloc<T>(out_);
     auto* output = out_->data<T>();
 
+#if defined _OPENMP
+#ifndef PADDLE_WITH_CUDA
+#pragma omp parallel for
+#endif
+#endif
```

Comment:
Why do you need these ifdefs here?
```suggestion
#pragma omp parallel for
```

Reply:
One CI was complaining about not being able to include the OMP library, so I'm checking if it's available. Also, the ops tests were failing on CI Coverage so I decided to disable GPU support.

Reply:
sounds like CMake mess